common/messages.go

```diff
@@ -33,9 +33,16 @@ type ConfigValidationMessage struct {
 
 type ConfigValidationResponse struct{}
 
+// View configuration for the UI display and optional tabs
+type View struct {
+	LayoutType      string   `json:"layout_type" msgpack:"layout_type"`
+	DefaultRequests []string `json:"default_requests" msgpack:"default_requests"`
```

Comment:
Did you want to remove the now-deprecated `default_requests` field from wherever it was before?

Reply:
ah yes i do, removed and added (assumed) changes to py